test/extensions/filters/http/ext_authz/ext_authz_fuzz_lib.cc

```diff
@@ -43,14 +41,8 @@ ReusableFilterFactory::newFilter(FilterConfigSharedPtr config,
 }
 
 absl::StatusOr<std::unique_ptr<Filter>> ReusableFuzzerUtil::setup(
-    const envoy::extensions::filters::http::ext_authz::ExtAuthzTestCase& input,
+    const envoy::extensions::filters::http::ext_authz::ExtAuthzTestCaseBase& input,
     Filters::Common::ExtAuthz::ClientPtr client) {
-  try {
```

Comment:
If you make setup() a template function, then you can continue share this common code.

Reply:
It should move out of the library anyway because the clients are currently using unvalidated proto values. For example in the grpc fuzz test, `newGrpcClientImpl` is called on the unvalidated proto.